python/py-musicbrainzngs/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        alastair python-musicbrainzngs 0.7.1 v
```

Comment:
we would typically prefer to get the sources from PyPI. You don't need the `github` PG and you would just do:
```
name                py-musicbrainzngs
version             0.71
```

but I see that for most of the ones you submitted so far you're using the GitHub sources... there isn't necessarily anything wrong with it so we can leave it as-as. But just for future reference.

Reply:
@reneeotten let me when move away from github when at all this PR :)